tests/unit/test.config.js

```diff
@@ -927,6 +960,7 @@ describe('config', () => {
     it('finds a config in your home directory', () => {
       return withTempDir(
         async (tmpDir) => {
+          const packageJSON = path.join(process.cwd(), 'package.json');
```

Comment:
Instead of editing this test, you should make a new one. The test you edited is specific to checking that a config in your home directory gets loaded. You need a new test specific to checking that a `package.json` file gets loaded. The test will probably look more like [this one](https://github.com/ChujunGeng/web-ext/blob/204fed982bd8333a25dd33c6b19de17c6374d77f/tests/unit/test.config.js#L979).

Reply:
I edited this test because web-ext itself's `package.json` will be loaded (apparently because it's inside current working directory), and that's not expected by the original test. Unless we cd to a temporary directory, and make another temporary directory home directory to test out `.web-ext-config.js`, this test will always fail for the aforementioned reason.

Reply:
Oh, I see. It would be helpful if you can add a comment to this test explaining the situation.